pkg/kritis/admission/admission.go

```diff
@@ -132,7 +132,8 @@ func checkBreakglass(pod *v1.Pod) bool {
 	if annotations == nil {
 		return false
 	}
-	_, ok := annotations[constants.BREAKGLASS]
+	breakglass := fmt.Sprintf("%s/%s", kritis.GroupName, constants.BREAKGLASS)
```

Comment:
Might be nice to make all of these labels be constants themselves, declared somewhere globally.

Reply:
done